app/components/gh-members-lab-setting.js

```diff
@@ -26,9 +26,14 @@ export default Component.extend({
     config: service(),
     mediaQueries: service(),
 
-    currencies: currencies,
+    currencies: null,
 
-    selectedCurrency: null,
+    // passed in actions
+    setMembersSubscriptionSettings() {},
+
+    selectedCurrency: computed('subscriptionSettings.stripeConfig.plans.monthly.currency', function () {
+        return CURRENCIES.findBy('value', this.get('subscriptionSettings.stripeConfig.plans.monthly.currency'));
```

Comment:
Using a CP here means that we can select the right object from our currencies array automatically any time our config changes without needing to manually set it anywhere.